### PR TITLE
Remove TODO and implemented feature in ER creation

### PR DESCRIPTION
**Pull request by anandjakhaniya**

### CODE REVIEW

File: backend/src/main/java/ai/verta/modeldb/experimentRun/FutureExperimentRunDAO.java [hunk 1]
```diff
@@ -628,8 +631,11 @@ public FutureExperimentRunDAO(Executor executor, FutureJdbi jdbi, UAC uac) {
                 observationHandler.logObservations(
                     newExperimentRun.getId(), newExperimentRun.getObservationsList(), now),
             executor)
-        // TODO .thenCompose(handle -> featureHandler.logFeatures(newExperimentRun.getId(),
-        // newExperimentRun.getFeaturesList()), executor)
+        .thenCompose(
+            handle ->
+                featureHandler.logFeatures(
+                    newExperimentRun.getId(), newExperimentRun.getFeaturesList()),
+            executor)
```
Comment (conradoverta) [inline backend/src/main/java/ai/verta/modeldb/experimentRun/FutureExperimentRunDAO.java]: Same feedback as main PR

**Review by conradoverta — CHANGES_REQUESTED** [review]
Are features like tags?

**Comment by anandjakhaniya** [comment]
> Are features like tags?

Yes it is.

File: backend/src/main/java/ai/verta/modeldb/experimentRun/FutureExperimentRunDAO.java (hunk 1 above)
Comment (anandjakhaniya) [reply backend/src/main/java/ai/verta/modeldb/experimentRun/FutureExperimentRunDAO.java]: Fixed

**Review by conradoverta — APPROVED** [review]

**Pull request merged**